### PR TITLE
Feature/header

### PR DESCRIPTION
**Pull request by ghost**
## Summary

* Customize navigation header

#### Galaxy Nexus
image

#### Pixel 3XL
image

#### iPhone 8
image

#### iPhone 12 Pro Max
image

## Trello Card

[Header](https://trello.com/c/JjT39mhc/15-n-74-header)

### CODE REVIEW

File: wbooks/src/app/components/AppNavigator/index.tsx [hunk 1]
```diff
@@ -2,17 +2,29 @@ import React from 'react';
 import { createStackNavigator } from '@react-navigation/stack';
 import LibraryScreen from '@screens/Library';
 import BookDetailScreen from '@screens/BookDetail';
+import HeaderBackground from '@app/components/HeaderBackground';
+import HeaderBackButton from '@app/components/HeaderBackButton';
```
Comment (sfernandez11) [inline wbooks/src/app/components/AppNavigator/index.tsx]: `@app/components` => `@components`

File: wbooks/src/app/components/HeaderBackButton/index.tsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { Image } from 'react-native';
+import icBack from '@assets/general/ic_back.png';
+
+import styles from './styles';
+
+function HeaderBackButton() {
+  return <Image style={styles.container} source={icBack} />;
```
Comment (sfernandez11) [inline wbooks/src/app/components/HeaderBackButton/index.tsx]: And the `resizeMode`? Maybe `cover`?

File: wbooks/src/app/components/HeaderBackground/index.tsx [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { Image } from 'react-native';
+import bcImage from '@assets/general/bc_nav_bar.png';
+
+import styles from './styles';
+
+function HeaderBackground() {
+  return <Image style={styles.image} source={bcImage} />;
```
Comment (sfernandez11) [inline wbooks/src/app/components/HeaderBackground/index.tsx]: Same here, don't forget about `resizeMode`. Maybe `resizeMode="stretch"`?

File: wbooks/src/app/components/HeaderBackground/styles.ts [hunk 4]
```diff
@@ -0,0 +1,7 @@
+import { StyleSheet } from 'react-native';
+
+export default StyleSheet.create({
+  image: {
+    width: '100%'
```
Comment (sfernandez11) [inline wbooks/src/app/components/HeaderBackground/styles.ts:7]: Missing `flex` (I think, test if it's necessary or not) and `backgroudColor`

**Review by sfernandez11 — CHANGES_REQUESTED** [review]
Great work, some comments...

File: wbooks/src/app/components/HeaderBackButton/index.tsx (hunk 2 above)
Comment (killermu94) [reply wbooks/src/app/components/HeaderBackButton/index.tsx:8]: I tried with some combinations using resizeMode, but the result was the same

File: wbooks/src/app/components/HeaderBackground/index.tsx (hunk 3 above)
Comment (ghost) [reply wbooks/src/app/components/HeaderBackground/index.tsx]: I tried with all combinations using `resizeMode`, but the result was the same

File: wbooks/src/app/components/HeaderBackButton/index.tsx (hunk 2 above)
Comment (ghost) [reply wbooks/src/app/components/HeaderBackButton/index.tsx]: Mm if I add `resizeMode="cover"`, I can not see any change

File: wbooks/src/app/components/HeaderBackground/styles.ts (hunk 4 above)
Comment (ghost) [reply wbooks/src/app/components/HeaderBackground/styles.ts:7]: If I added `flex: 1`, this convert the `height` to `90`, without `flex: 1`, the image `height` is `102`

File: wbooks/src/app/components/AppNavigator/index.tsx (hunk 1 above)
Comment (ghost) [reply wbooks/src/app/components/AppNavigator/index.tsx]: Done!

File: wbooks/src/app/components/HeaderBackButton/index.tsx (hunk 2 above)
Comment (ghost) [reply wbooks/src/app/components/HeaderBackButton/index.tsx]: Done!

File: wbooks/src/app/components/HeaderBackground/index.tsx (hunk 3 above)
Comment (ghost) [reply wbooks/src/app/components/HeaderBackground/index.tsx]: Done!

**Review by sfernandez11 — APPROVED** [review]
🥇

**Pull request merged**